libyul/backends/evm/EVMDialect.cpp

```diff
@@ -137,6 +137,52 @@ set<YulString> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 	return reserved;
 }
 
+/// Minimal version of evmasm::InstructionInfo
+struct MinInstructionInfo
```

Comment:
Please move as much code as possible to another file like `zkEVMExtension.cpp`.
It will be easier to rebase changes and do updates this way.